BioArchLinux/igv/lilac.yaml

```diff
@@ -0,0 +1,14 @@
+build_prefix: extra-x86_64
+maintainers:
+- github: kiri2002
+  email: kiri@vern.cc
+pre_build_script: |
+  update_pkgver_and_pkgrel(_G.newver)
+  run_cmd(['updpkgsums'])
+post_build_script: |
+  git_pkgbuild_commit()
+update_on:
+  - source: github
+    github: igvteam/igv
+    use_latest_tag: true
```

Comment:
refer to [nvchecker's doc](https://nvchecker.readthedocs.io/en/latest/usage.html#check-github):

> use_latest_tag
> 
>     Set this to true to check for the latest tag on GitHub.
> 
>     This requires a token because it’s using the v4 GraphQL API.

I thinks "use_latest_tag" maybe more robust V.S. "use_max_tag", especially some times, it's history tag is bigger.